doc/tracking-stories-on-clubhouse.md

```diff
@@ -0,0 +1,7 @@
+# Tracking stories on Clubhouse
```

Comment:
Ok, I am _really_ sorry about this, but I think I got confused when the PR title referred to a change in the ADR. An ADR is a very specific type of documentation separate from our typical documentation (as can be seen in the `doc/arch` directory) and it needs to follow the conventions I mentioned earlier.

I just noticed that this documentation is not actually an ADR and is just a regular piece of documentation in which case your original set of changes was actually correct, except for the labelling of these changes as an ADR when it should just be documentation.

Could you please revert these changes and update the commit and PR messages to remove the references to "ADR"? I could make these changes as well if you're occupied.

Reply:
Oh okay. Understood. I'll update this